packages/tools/examples/stackRotateTool/index.ts

```diff
@@ -0,0 +1,121 @@
+import { RenderingEngine, Types, Enums } from '@cornerstonejs/core';
```

Comment:
I didn't mean to create a new example. I was talking about just having it in the stack manipulation tools, but don't make it active by default.

Reply:
Resolved, please check again. @sedghi